### DISCUSSION TITLE
Response from support team

### DISCUSSION THREAD

**Discussion by SergGrebennikov** [community/community#39749] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hey everyone, could someone give a hint on how to get the attention of the support team to my ticket? A week without an answer( and it is a blocker for us to proceed.

**Comment by queenofcorgis** [comment]
Hi there @SergGrebennikov - the GitHub Community is unable to assist in escalating support tickets - please be patient! Our Support team will assist you as soon as they are able to as they typically process tickets in the order they were received.
